packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -98,6 +99,9 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 		isEnabled( 'signup/design-picker-style-selection' ) && isEnableThemePreviewScreen;
 	const shouldUpgrade = isPremium && ! isPremiumThemeAvailable && ! hasPurchasedTheme;
 
+	const showBundledBadge =
```

Comment:
[nit] So that we don't confuse future devs working on this, can we make this more specific? I'm thinking of something in the lines of

```
const isBundledWithWooCommerce =
  isEnabled( 'themes/plugin-bundling' )
    && (design.taxonomies?.theme_software_set || []).some(({slug}) => slug === 'woo-on-plans');
```